ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -111,7 +116,7 @@ class LocalAssemblerData : public LocalAssemblerDataInterface<GlobalMatrix, Glob
 
 private:
     std::vector<ShapeMatrices> _shape_matrices;
-    double _hydraulic_conductivity;
+    std::function<double(void)> _hydraulic_conductivity;
```

Comment:
Here we need return matrix in future development for anisotropic conductivity. Everything else looks fine.